instrumentation/opentelemetry-instrumentation-asgi/src/opentelemetry/instrumentation/asgi/__init__.py

```diff
@@ -205,7 +206,7 @@ async def __call__(self, scope, receive, send):
                 @wraps(receive)
                 async def wrapped_receive():
                     with self.tracer.start_as_current_span(
-                        span_name + " asgi." + scope["type"] + ".receive"
+                        span_name + " " + scope["type"] + ".receive"
```

Comment:
`scope["type"].receive` is not defined in the specs. I am wondering what we should do with these spans.
@codeboten thoughts?

Reply:
im wondering if the SpanKind should be set here in addition to having the name of the span reflect the sender/receiver as per the messaging semantic conventions: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/semantic_conventions/messaging.md#span-name